src-1.2/bootstrap.js

```diff
@@ -53,6 +53,33 @@ async function waitForZotero() {
 	await Zotero.initializationPromise;
 }
 
+// Adds main window open/close listeners in Zotero 6
+function listenForMainWindowEvents() {
+	const mainWindowListener = {
+		onOpenWindow: function (aWindow) {
+			let domWindow = aWindow.QueryInterface(Ci.nsIInterfaceRequestor)
+				.getInterface(Ci.nsIDOMWindowInternal || Ci.nsIDOMWindow);
+			async function onload() {
+				domWindow.removeEventListener("load", onload, false);
+				if (domWindow.location.href !== "chrome://zotero/content/zoteroPane.xhtml") {
```

Comment:
@windingwind @dstillman, I've added this code to my plugin, but I'm noticing that the `href` for the main window is coming back as `chrome://zotero/content/standalone/standalone.xul` instead of the URI specified here. Is that expected?

Reply:
I submitted a proposed fix in #5.